Upstream's CMakeLists.txt hardcodes a stale version (2.1.0a) that does not
match the v1.9.1 release tag.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,9 +10,9 @@
 enable_language (Fortran)
 
 # The version number.
-set (Puffin_VERSION_MAJOR 2)
-set (Puffin_VERSION_MINOR 1)
-set (Puffin_VERSION_PATCH 0a)
+set (Puffin_VERSION_MAJOR 1)
+set (Puffin_VERSION_MINOR 9)
+set (Puffin_VERSION_PATCH 1)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules")
 
